src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
Can you post more captured messages? Or actual logic capture data if some tool to view them is available? Are there some delays in message?